packages/website/src/content/docs/sidecar/index.mdx

```diff
@@ -21,7 +21,7 @@ Also in the Sentry SDKs the URL can be configured via the `spotlight` option, fo
 import sentry_sdk
 
 sentry_sdk.init(
-    spotlight="http://localhost:8969/stream",
+    spotlight="http://localhost:8969",
```

Comment:
No no no definitely not. The SDKs expect the `/stream` part and at this point we should keep it that way unfortunately.

Reply:
yeah, but ideally we should just ask for sidecar url.
I know initially, we were only having stream url i.e. why i guess we asked for stream url directly.

let me think if we could do something to support /stream url as well here.

Reply:
@Shubhdeep12 I think a cheap hack is to see if we have the URL ending with `/stream` and try to guess from there.

That said not having the `/stream` part in the SDKs is a no go, as it requires us to change all existing SDKs to support this _and_ have a fallback or cut a major release. Don't want either so that ship seems to have sailed.

Reply:
No, I wasn't intending to remove the /stream path. I meant that the user should focus on just sending the sidecarURL while initializing Spotlight. We'll handle which path to use for different cases, like:

- /stream for EventSource,
- /clear for clearing all events,
- /open for opening the editor, etc.

Reply:
Can we do this-
- Remove the /stream suffix from sidecar URL passed in Spotlight.init
- Store the sidecar url
- and then using /stream, /open, /clear ,etc based on the need in the code

Reply:
Just pushed this, let me know your thoughts on this.